uisdk/src/main/res/values-en-rGB/strings.xml

```diff
@@ -214,6 +215,10 @@
     <string name="kh_uisdk_last_name_hint">Last name</string>
     <string name="kh_uisdk_last_trip_id">Last Trip ID: %1$s</string>
     <string name="kh_uisdk_last_trip_id_not_set">Last Trip ID: Not yet set</string>
+    <string name="kh_uisdk_legal_notice_label">Legal Notices</string>
```

Comment:
In iOS we have "Legal notice" label. I'm not saying it's good there, we have to decide and choose one form.